src/swap/SwapReviewScreen.tsx

```diff
@@ -148,7 +148,10 @@ export function SwapReviewScreen() {
 
   return (
     <SafeAreaView style={styles.safeAreaContainer}>
-      <CustomHeader title={t('swapReviewScreen.title')} left={<BackButton />} />
+      <CustomHeader
+        title={t('swapReviewScreen.title')}
+        left={<BackButton style={{ paddingHorizontal: variables.contentPadding }} />}
```

Comment:
Why is the padding different here? Now I'm convinced we should centralize that logic.

Reply:
Updated in c4fd878f8760e1507784e5888c8e8086bd4eba2b!